app/controllers/test/components/mentoring/mentor_inboxes_controller.rb

```diff
@@ -1,3 +1,9 @@
 class Test::Components::Mentoring::MentorInboxesController < ApplicationController
-  def show; end
+  def show
+    @sort_options = [
+      { value: 'recent', text: 'Sort by Most Recent' },
```

Comment:
`label` instead of `text` is more idiomatic, I think.

You can consider making the values live on some constant, so that you don't need to "match strings" but it's fine for now.

Reply:
(My suggestion would be a Query class, that encapsulates the logic for the mentor inbox query)